scripts/hooks/commit-msg.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs")
```

Comment:
Sanity check: can you confirm that this file's content is all original content? i.e. nothing copied from somewhere else?

Reply:
Yes, the COMMIT_MSG.txt (file path passed as argv aka HUSKY_PARAMETER) is the original file that contains the message for commit.

Reply:
What I meant is: did you author the whole of this file's content yourself?

Reply:
Ah, sorry, yes !